keyboards/0xcb/static/keymaps/via/keymap.c

```diff
@@ -0,0 +1,197 @@
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+// clang-format off
+enum layer_names {
+  _HOME,
+  _FN2,
+  _FN3,
+  _FN4
+};
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_HOME] = LAYOUT(
+                                                                                                       KC_MPLY,
+    KC_GESC, KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
+    KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,             KC_ENT,
+    KC_LSPO,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_RSPC,
+    KC_LCTL, KC_LALT, KC_LGUI, KC_SPC,           KC_SPC,                    KC_SPC,  KC_RALT, MO(1),   MO(2)
+),
+[_FN2] = LAYOUT(
+                                                                                                       RESET,
+    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,
+    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,           _______,
+    _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______,          _______,                   _______, _______, _______, _______
+),
+[_FN3] = LAYOUT(
+                                                                                                       EEP_RST,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LBRC, KC_RBRC, KC_BSLS,
+    _______, _______, _______, _______, _______, _______, _______, _______, KC_SCLN, KC_QUOT,          _______,
+    _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_SLSH,
+    _______, _______, _______, _______,          _______,                   _______, _______, _______, _______
+),
+[_FN4] = LAYOUT(
+                                                                                                       _______,
+    _______, KC_MUTE, KC_VOLU, KC_VOLD, KC_MPRV, KC_MNXT, KC_MSTP, KC_INS,  KC_HOME, KC_DEL,  KC_END,  _______,
+    _______, _______, KC_BRIU, KC_BRIU, _______, _______, _______, KC_PGUP, KC_UP,   KC_PGDN,          _______,
+    _______,          _______, _______, _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_RGHT, _______,
+    _______, _______, _______, _______,          _______,                   _______, _______, _______, _______
+),
+};
+// clang-format on
+
+/*layer switcher */
+uint32_t layer_state_set_user(uint32_t state) {
```

Comment:
Also, via has keycodes to handles this, actually. 
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```

Specifically `FN_MO13` and `FN_MO23`